linkerd/app/core/src/svc.rs

```diff
@@ -284,6 +284,14 @@ impl<S> Stack<S> {
         self.push(http::boxed::response::Layer::new())
     }
 
+    pub fn box_new<T>(self) -> Stack<stack::BoxNewService<T, S::Service>>
```

Comment:
nit/TIOLI: i might prefer to call this `box_new_service`...though it seems like we're not super consistent with naming the methods for new_service layers....